_includes/main_menu.html

```diff
@@ -1,144 +1,165 @@
+            <span class="dropdown-title">Community Stories</span>
+            <p>Learn how our community solves real, everyday machine learning problems with PyTorch</p>
           </a>
         </div>
       </div>
     </li>
 
     {% assign resources = "resources, features, hub, events, community-stories" | split: ", " %}
 
-    <li class="main-menu-item {% if resources contains current[1] %}active resources-active{% endif %}">
-
+    <li class="main-menu-item">
       <div id="resourcesDropdownButton" data-toggle="resources-dropdown" class="resources-dropdown">
-        <a class="resource-option {% if (current[1] == {{ site.baseurl }} or current[1] == 'foundation' or current[1] == 'join' or current[1] == 'edge' or current[1] == 'executorch-overview') %}with-down-white-arrow{% else %}with-down-arrow{% endif %}">
-          Resources
+        <a class="with-down-arrow">
+          About
         </a>
         <div class="resources-dropdown-menu">
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/features">
-            <span class=dropdown-title>About</span>
-            <p>Learn about PyTorch’s features and capabilities</p>
-          </a>
           <a class="nav-dropdown-item" href="{{ site.baseurl }}/foundation">
             <span class=dropdown-title>PyTorch Foundation</span>
             <p>Learn more about the PyTorch Foundation.</p>
           </a>
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/#community-module">
-            <span class=dropdown-title>Community</span>
-            <p>Join the PyTorch developer community to contribute, learn, and get your questions answered.</p>
-          </a>
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/community-stories">
-            <span class=dropdown-title>Community stories</span>
-            <p>Learn how our community solves real, everyday machine learning problems with PyTorch</p>
-          </a>
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/resources">
-            <span class=dropdown-title>Developer Resources</span>
-            <p>Find resources and get questions answered</p>
-          </a>
           <a class="nav-dropdown-item" href="{{ site.baseurl }}/events">
             <span class=dropdown-title>Events</span>
             <p>Find events, webinars, and podcasts</p>
           </a>
-          <a class="nav-dropdown-item" href="{{ site.external_urls.discuss }}" target="_blank">
-            <span class=dropdown-title>Forums</span>
-            <p>A place to discuss PyTorch code, issues, install, research</p>
-          </a>
-          <a class="nav-dropdown-item" href="{{ site.baseurl }}/hub">
-            <span class=dropdown-title>Models (Beta)</span>
-            <p>Discover, publish, and reuse pre-trained models</p>
-          </a>
         </div>
       </div>
     </li>
 
+    <li class="main-menu-item">
+      <a href="{{ site.baseurl }}/join" data-cta="join">
```

Comment:
If you hover over Join and GitHub it uses a cursor while other items use arrow - we should have it consistently.

Reply:
"Join" and the GitHub icon are clickable while the other top menu items are not that's why the mouse pointer behaves this way.